debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetricsMXBean.java

```diff
@@ -104,4 +104,55 @@
      * @param increment true to add, false to deduct
      */
     void changeSleepingTime(boolean increment);
+
+    void changeBatchSize(boolean increment);
+
+    /**
+     * this flag turns on recording of captured incremental changes. It creates an overhead on CPU and takes disk space
+     * @param doRecording true - record
+     */
+    void setRecordMiningHistory(boolean doRecording);
+
+    // record log mining history flag
+    boolean getRecordMiningHistory();
+
+    // *** following metrics work if RecordMiningHistory is true.
```

Comment:
Since the goal is to provide the `HistoryRecorder` as a hook rather than an implementation upstream, I think these metrics might make sense that they be exposed elsewhere as part of another JMX bean.  @jpechane would you agree?

Reply:
After reviewing these metrics, I'm curious why these are never actually used in the runtime code, @AndreyIg?  I see we have a test case that checks that the method calls for these manipulate the metrics state accordingly; however, we actually never track these at runtime.

Reply:
agree, we should eliminate those

Reply:
@Naros +1